controllers/podchaos/types.go

```diff
@@ -40,7 +46,37 @@ func (r *Reconciler) Reconcile(req ctrl.Request) (ctrl.Result, error) {
 		r.Log.Error(err, "unable to get podchaos")
 		return ctrl.Result{}, nil
 	}
+	scheduler := podchaos.GetScheduler()
+	duration, err := podchaos.GetDuration()
+	if err != nil {
+		r.Log.Error(err, fmt.Sprintf("unable to get podchaos[%s/%s]'s duration", podchaos.Namespace, podchaos.Name))
+		return ctrl.Result{}, nil
+	}
+	if scheduler == nil && duration == nil {
```

Comment:
How about `pod-kill` action? `Duration` is always empty for `pod-kill`

Reply:
good catch and updated.